graylog2-web-interface/src/components/common/TimezoneSelect.jsx

```diff
@@ -12,25 +12,33 @@ const TimezoneSelect = React.createClass({
   getValue() {
     return this.refs.timezone.getValue();
   },
+
+  // Some time zones are not stored into any areas, this is the group we use to put them apart in the dropdown
+  // https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+  _UNCLASSIFIED_AREA: 'Unclassified',
+
   _formatTimezones() {
     const timezones = {};
     moment.tz.names().forEach((timezone) => {
```

Comment:
@joschi I think that would only move the problem to another place: when we use the time zone to convert UTC times into local times, as that is mostly done in the frontend nowadays. So, in the end, moment.js and Joda-Time need to support the same time zones, or we need to have a layer between them that does some conversion for us. So far I didn't find any example that breaks one of the libraries, so I assume they do use the same names currently.